### ISSUE TITLE
Text formatting needed in the CONTRIBUTIONS.md file

### ISSUE THREAD

**Issue by donmarvex** [zero-to-mastery/start-here-guidelines#15702]
### Description
I went through the list of contributors in the CONTRIBUTIONS.md file and couldn't help but notice some deviations from the initial format used for making entries of usernames/links to profiles.

The required
```Markdown
- [@Username](profile link)
``` 

Deviations
```Markdown
-[@Username](profile link). 
# No whitespace after the hyphen "-"

-[@Username] (profile link). 
# Whitespace between ] and (
```

### Possible Resolutions?
- Manually remove whitespace from a code editor across all lines which have this discrepancy
- Possibly an automated solution (which will become a CI action whenever a push action occurs) to run through the file and remove/add whitespace where needed.

If anyone is interested, you could indicate to get assigned.

**Comment by 7ORP3DO** [comment]
Hii @donmarvex,  I want to work on this issue.

**Comment by donmarvex** [comment]
> Hii @donmarvex, I want to work on this issue.

Hello @7ORP3DO , I will now assign you to work on this. When you are done just make a pull request referencing this issue and tag me to it so we can do a review to see that the problem is solved.

Thank You.

**Comment by donmarvex** [comment]
The issue has been resolved with the PR #15716
